builder/virtualbox/ovf/config_test.go

```diff
@@ -30,24 +30,6 @@ func getTempFile(t *testing.T) *os.File {
 	return tf
 }
 
-func testConfigErr(t *testing.T, warns []string, err error) {
```

Comment:
probably shouldn't get rid of these helper functions, since they aid readability. can probably continue to use them even if the tests have changed